k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala

```diff
@@ -113,11 +113,9 @@ class EndpointsNamer(idPrefix: Path, mkApi: String => NsApi) extends Namer {
 private object EndpointsNamer {
   val PrefixLen = 3
 
-  type VarUp[T] = Var[T] with Updatable[T]
-  type ActUp[T] = VarUp[Activity.State[T]]
```

Comment:
These type aliases also exist in CatalogNamer.scala -- do you think it's worth trying to remove them from there as well?